my-app/src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
```

Comment:
Consider separating third-party imports from the project's ones.

Reply:
/fxd